addons/udes_warehouse_classification/models/stock_picking.py

```diff
@@ -14,7 +14,8 @@ def _get_classification_messages_for_product_picking(self):
                 lambda c: self.picking_type_id in c.picking_type_ids
             ).sorted("sequence")
             product_classifications[product.barcode] = [
-                {"message": classification.alert_message} for classification in classifications
+                {"message": classification.alert_message, "name": classification.name}
```

Comment:
I think we just need the alert message and not the name of the classification. So we display the `alert_message` in the mobile UI.